examples/optimize_evoked.py

```diff
@@ -0,0 +1,79 @@
+"""
+=================================================
+08. Optimize simulated evoked response parameters
+=================================================
+
+This example demonstrates how to optimize the parameters
+of the model simulation to match an experimental dipole waveform.
+"""
+
+# Authors: Blake Caldwell <blake_caldwell@brown.edu>
+#          Mainak Jas <mjas@mgh.harvard.edu>
+
+import os.path as op
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+###############################################################################
+# Let us import hnn_core
+
+import hnn_core
+from hnn_core.dipole import simulate_dipole
+from hnn_core import read_params, Dipole, MPIBackend, Network
+
+hnn_core_root = op.join(op.dirname(hnn_core.__file__))
+
+mpi_cmd = '/autofs/space/meghnn_001/users/mjas/opt/openmpi/bin/mpirun'
```

Comment:
Is there a way to automatically set this?

Reply:
this is just set like this for my environment because I compiled it on the server. Couldn't figure out how to get it to find my compiled version automatically ... I'm guessing it works for you without having to set this?

Reply:
Oh okay. Yes, I can just enter `mpiexec` or `mpirun` and everything works fine.